app/renderer/components/tabs/tabPages.js

```diff
@@ -3,39 +3,43 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const React = require('react')
+const Immutable = require('immutable')
 
 // Components
-const ImmutableComponent = require('../immutableComponent')
+const ReduxComponent = require('../reduxComponent')
 const TabPage = require('./tabPage')
 
+ // Constants
+const settings = require('../../../../js/constants/settings')
+
 // Utils
-const dnd = require('../../../../js/dnd')
+const frameStateUtil = require('../../../../js/state/frameStateUtil')
+const {getSetting} = require('../../../../js/settings')
+
+class TabPages extends React.Component {
+  mergeProps (state, ownProps) {
+    const currentWindow = state.get('currentWindow')
+    const frames = frameStateUtil.getNonPinnedFrames(currentWindow) || Immutable.List()
+    const tabsPerPage = Number(getSetting(settings.TABS_PER_PAGE))
+
+    const props = {}
+    props.tabPageCount = Math.ceil(frames.size / tabsPerPage)
```

Comment:
ahh that's right! I had forgotten 😄  I saw you use getters in https://github.com/brave/browser-laptop/pull/9413, but I now see those are in `render`.  Util method would be great 😄